examples/simple_box/src/server.rs

```diff
@@ -17,17 +17,26 @@ use std::sync::Arc;
 use crate::protocol::*;
 use crate::shared;
 
+
+
 pub struct ExampleServerPlugin;
 
+
 impl Plugin for ExampleServerPlugin {
     fn build(&self, app: &mut App) {
+        app.init_resource::<ClientEntityMap>();
         app.add_systems(Startup, (init, start_server));
-        // the physics/FixedUpdates systems that consume inputs should be run in this set
```

Comment:
can we keep thsi comment? it's pretty important

Reply:
Yes